patches/0005-Add-CNG-crypto-backend.patch

```diff
@@ -225,11 +223,6 @@ index 00000000000000..39c3c7043ab720
 +			panic("cngcrypto: not in FIPS mode")
 +		}
 +	}
-+	if enabled, _ := cng.FIPS(); enabled {
-+		// FIPS mode is enabled,
-+		// so force FIPS mode for crypto/tls and crypto/x509.
-+		fipstls.Force()
```

Comment:
There doesn't seem to be any mechanism for CNG's FIPS mode to force FIPS TLS anymore. (Tested on my Windows machine, and non-FIPS algorithms do show up where they don't with 1.23.)

Reply:
Good catch. I'll manually call `BCryptGetFipsAlgorithmMode` in CNG's `systemFIPSMode` implementation so that we get back the previous behavior.